README.md

```diff
@@ -71,4 +94,39 @@ def main:
     print (2 + 2)
 ```
 
-To compile and run the project, simply type `luna` in the main project directory.
+To compile and run the project, simply type `luna` in the main project 
+directory.
+
+## Contributing to Luna
+If you are interested in contributing to the development of Luna, please read
+the 
+[`CONTRIBUTING.md`](https://github.com/luna/luna/blob/master/CONTRIBUTING.md)
+file. 
+
+## License
+This repository is licensed under the
+[Apache 2.0](https://opensource.org/licenses/apache-2.0), as specified in the
+[LICENSE](https://github.com/luna/luna/blob/master/LICENSE) file. All
+contributions to this repository are accepted under these terms.
```

Comment:
Again, since we have a CLA in place, I think this additional constrain is not required.